ffmpeg/build_ci.sh

```diff
@@ -20,14 +20,22 @@ git fetch origin
 FOLDER_COMMIT=$(git log -1 --format=format:%H --full-diff $DIR)
 BASE_COMMIT=$(git show-ref refs/remotes/origin/master | awk '{print $1}')
 
+git merge-base --is-ancestor "${FOLDER_COMMIT}" "${BASE_COMMIT}"
+COMMIT_SAME=$?
+
 set -exou pipefail
 
 mkdir -p "${DIR}/output/"
 # This is so that circle's persist_to_workspace doesn't fail when builds don't run
 touch "${DIR}/output/hello"
 
-if git merge-base --is-ancestor "${FOLDER_COMMIT}" "${BASE_COMMIT}"; then
+if [ $COMMIT_SAME -eq 1 ]; then
```

Comment:
The previous command should just work, I don't think this change is actually necessary

Reply:
I tried to run it both locally and on the CI on the previous version, and it didn't work at all, I think it is related to the `set -exou` that comes after assigning the variable

Reply:
I definitely thought conditionals were exempt from the `set -e` erroring out, but I'm willing to just push this through since it's relatively small.